packages/core/src/components/select/stories/select.core.stories.tsx

```diff
@@ -32,13 +32,19 @@ const createSelect = (args) => {
         atomSelectElements.forEach((atomSelect) => {
           atomSelect.options = [
             { id: '1', value: 'Red', disabled: false },
-            { id: '2', value: 'Green', disabled: false },
+            {
+              id: '2',
+              value: 'Green',
+              disabled: false,
+              tag: { color: 'success', label: 'New' },
```

Comment:
block: taking into account that not every select will have a badge, we need to create a new session in Storybook with just this behavior

Reply:
What do you think? https://github.com/juntossomosmais/atomium/pull/549/commits/373e09dd5e2323ed5811788ba437016cd7fa8ccf